python_ta/config/messages_config.toml

```diff
@@ -72,6 +72,7 @@ W0301 = "You should remove this semicolon. In Python, statements are separated b
 C0301 = "This line is %s characters long, exceeding the limit of %s characters."
 C0303 = "Trailing whitespace is poor formatting, and should be removed."
 C0304 = "By convention, each Python file should end with a blank line. To fix this, go down to the bottom of your file and press Enter/Return to add an empty line."
+C0305 = "Trailing newlines are poor formatting, and should be removed."
```

Comment:
I think we can reword this to align with C0304. "By convention, each Python file should end with exactly one blank line. To fix this, go down to the bottom of your file and delete the extra blank lines so that there is only one left."